src/TankControllerLib.h

```diff
@@ -3,6 +3,7 @@
 
 #include "Devices/DateTime_TC.h"
 #include "Devices/EthernetServer_TC.h"
+#include "Devices/Keypad_TC.h"
```

Comment:
Is this needed at this time?

Reply:
Upon moving it to TankControllerLib.cpp the tests wouldn't compile anymore. Is there anywhere else I should put the include instead?